app/controllers/api/service_templates_controller.rb

```diff
@@ -4,5 +4,56 @@ class ServiceTemplatesController < BaseController
     include Subcollections::Tags
     include Subcollections::ResourceActions
     include Subcollections::ServiceRequests
+
+    def create_resource(_type, _id, data)
```

Comment:
Shouldn't the method be named `create_service_template` unless `resource` is a term used by the API